docs/examples/ControllerExample/loom.config

```diff
@@ -0,0 +1,15 @@
+{
+  "sdk_version": "dev",
```

Comment:
Oh, and the art was not done by me, except for the ugly trigger button gauges. Mostly because I wanted to save on time.
I got it from here http://txusmetal4ever.deviantart.com/art/Vector-PlayStation-gamepad-441805194
>PSD download available. Anyone's free to edit and/or use it. No credits are needed.